src/containers/Projects/styles.module.scss

```
@@ -1,8 +1,13 @@
 @import '../../theme/variables.scss';
+@import '../../theme/globals.scss';
 
 .featured {
 }
 
+.heading__main {
+  @extend h1;
```

If you import globals then you also import all the classes there, which creates duplication when used with CSS modules. Since globals are already imported with the app core, you can compose global classes. 

```suggestion
  composes: h1 from global;
```